pulumi/volumeReaper/src/index.ts

```diff
@@ -1,13 +1,12 @@
 import figlet from 'figlet'
 import { Command } from 'commander'
-import { runBackup } from './backup'
+import { VolumeReaper } from './volumeReaper'
 
 console.log(figlet.textSync('Unchained Volume Reaper'))
 
 const program = new Command()
 
 program
-  .version('1.0.0')
```

Comment:
thought it would be hard to keep in line with package version, though we will probably never bump, not really relevant either way